app/models/embeddable/open_response.rb

```diff
@@ -51,5 +51,6 @@ def export
     def self.import (import_hash)
       return self.new(import_hash)
     end
+    include CRater::SettingsProviderFunctionality
```

Comment:
Please add a comment above this line explaining why it needs to be included after the methods are defined.